src/Nethermind/Chains/chiado.json

```diff
@@ -40,11 +40,14 @@
     }
   },
   "params": {
+    "blobGasPriceUpdateFraction": "0x10fafa",
```

Comment:
very minor: maybe keep cancun values close to each other rather than alphabetic?

Reply:
I don't think it brings much value, as not everyone knows what parameter belongs to EIP-4844 or Cancun while it's always easy to check visually whether a particular parameter is defined.